test/e2e/app-dir/app-css/index.test.ts

```diff
@@ -50,16 +50,17 @@ createNextDescribe(
           )
         })
 
-        it('should support external css imports', async () => {
+        it.only('should support external css imports', async () => {
           const browser = await next.browser('/css/css-external')
+          await new Promise(() => {})
           await check(
             async () =>
               await browser.eval(
                 `window.getComputedStyle(document.querySelector('main')).paddingTop`
               ),
             '80px'
           )
-        })
+        }, 1_000_000_000)
```

Comment:
This is a gigantic timeout value. If this does need a timeout, should it be lower?

Reply:
Sorry, the changes in this file were debugging.